packages/openapi-ts/test/__snapshots__/test/generated/v3_models/models.ts.snap

```diff
@@ -722,6 +722,10 @@ export type ModelWithReadOnlyAndWriteOnly = {
     baz: string;
 };
 
+export type ModelWithConstantSizeArray = [number];
```

Comment:
Shouldn't this produce

```suggestion
export type ModelWithConstantSizeArray = [number, numer];
```

since minItems and maxItems is 2?

Reply:
Will be fixed in other PR